src/components/pages/DAOcreator/SchemesStep/AddSchemeDialog.tsx

```diff
@@ -93,23 +98,51 @@ class VerticalLinearStepper extends React.Component<Props, State> {
     }))
   }
 
-  addOrUpdateSchemeParam = async (paramName: string, value: any) => {
+  addOrUpdateSchemeParam = async (
+    paramName: string,
+    value: any,
+    valueType: string
+  ) => {
     const { id, typeName, params } = this.state.schemeConfig
+    let errorMessage = ""
+
+    switch (valueType) {
```

Comment:
I think we could use the params validation function here. But it might not make sense to check for other things then type? I am not sure 😛 If so the validation function might be redundant?

We will, either way, need to do context-aware validation in here.

@gh1dra what do you think we should do?